scripts/benchmarks-ci.sh

```diff
@@ -22,28 +22,53 @@ if [[ $runtimeName == "statemint" ]] || [[ $runtimeName == "statemine" ]] || [[
 		pallet_uniques
 		cumulus_pallet_xcmp_queue
 		frame_system
+	)
+	xcm_pallets=(
 		pallet_xcm_benchmarks::generic
 		pallet_xcm_benchmarks::fungible
 	)
 elif [[ $runtimeName == "collectives-polkadot" ]]; then
-		pallets=(
-			pallet_alliance
-			pallet_balances
-			pallet_collator_selection
-			pallet_collective
-			pallet_multisig
-			pallet_proxy
-			pallet_session
-			pallet_timestamp
-			pallet_utility
-			cumulus_pallet_xcmp_queue
-			frame_system
-		)
+	pallets=(
+		pallet_alliance
+		pallet_balances
+		pallet_collator_selection
+		pallet_collective
+		pallet_multisig
+		pallet_proxy
+		pallet_session
+		pallet_timestamp
+		pallet_utility
+		cumulus_pallet_xcmp_queue
+		frame_system
+	)
+	xcm_pallets=(
+	)
 else
 	echo "$runtimeName pallet list not found in benchmarks-ci.sh"
 	exit 1
 fi
 
+for pallet in ${xcm_pallets[@]}
+do
+	# a little hack for xcm benchmarks
+	output_file="${pallet//::/_}"
+  if [[ "$pallet" == *"xcm"* ]]; then
+		output_file="xcm/$output_file"
+  fi
```

Comment:
this probably not necessary, they all xcm.
and the loop below also can be simplified

Reply:
actually let's lean into that and use that for setting the template.